tests/e2e/ctl_v3_auth_test.go

```diff
@@ -1326,3 +1328,89 @@ func ctlV3User(cx ctlCtx, args []string, expStr string, stdIn []string) error {
 	_, err = proc.Expect(expStr)
 	return err
 }
+
+// authTestCacheReload tests the permissions when a member restarts
+func authTestCacheReload(cx ctlCtx) {
+
+	authData := []struct {
+		user string
+		role string
+		pass string
+	}{
+		{
+			user: "root",
+			role: "root",
+			pass: "123",
+		},
+		{
+			user: "user0",
+			role: "role0",
+			pass: "123",
+		},
+	}
+
+	node0 := cx.epc.Procs[0]
+	endpoint := node0.EndpointsV3()[0]
+
+	// create a client
+	c, err := clientv3.New(clientv3.Config{Endpoints: []string{endpoint}, DialTimeout: 3 * time.Second})
+	if err != nil {
+		cx.t.Fatal(err)
+	}
+	defer c.Close()
+
+	for _, authObj := range authData {
+		// add role
+		_, err = c.RoleAdd(context.TODO(), authObj.role)
+		if err != nil {
+			cx.t.Fatal(err)
+		}
+
+		// add user
+		_, err = c.UserAdd(context.TODO(), authObj.user, authObj.pass)
+		if err != nil {
+			cx.t.Fatal(err)
+		}
+
+		// grant role to user
+		_, err = c.UserGrantRole(context.TODO(), authObj.user, authObj.role)
+		if err != nil {
+			cx.t.Fatal(err)
+		}
+	}
+
+	// role grant permission to role0
+	_, err = c.RoleGrantPermission(context.TODO(), authData[1].role, "foo", "", clientv3.PermissionType(clientv3.PermReadWrite))
+	if err != nil {
+		cx.t.Fatal(err)
+	}
+
+	// enable auth
+	_, err = c.AuthEnable(context.TODO())
+	if err != nil {
+		cx.t.Fatal(err)
+	}
```

Comment:
```suggestion
	if _, err = c.AuthEnable(context.TODO()); err != nil {
		cx.t.Fatal(err)
	}
```

This is just a minor comment, but it can improve the readaiblity a little bit. The same to following couple of places.

Reply:
Addressed, thanks.